gcloud/logging/client.py

```diff
@@ -47,6 +49,28 @@ class Client(JSONClient):
     """
 
     _connection_class = Connection
+    _logging_api = _sinks_api = _metrics_api = None
+
+    @property
+    def logging_api(self):
+        """Helper for logging-related API calls."""
+        if self._logging_api is None:
+            self._logging_api = _LoggingAPI(self.connection)
+        return self._logging_api
+
+    @property
+    def sinks_api(self):
+        """Helper for logging-related API calls."""
+        if self._sinks_api is None:
+            self._sinks_api = _SinksAPI(self.connection)
+        return self._sinks_api
+
+    @property
+    def metrics_api(self):
+        """Helper for logging-related API calls."""
```

Comment:
Starting my docstring copy-pasta check. You have the same message for all three of these properties. Make it more distinct (to identify which helper)?